Integrations/integration-Carbon_Black_Enterprise_Response.yml

```diff
@@ -330,7 +330,7 @@ script:
         if (facets) {
             queryParams.facets = facets;
         }
-        let response = sendRequest('/v1/process', 'GET', encodeToURLQuery(queryParams));
+        let response = sendRequest('/v2/process', 'GET', encodeToURLQuery(queryParams));
```

Comment:
searching processes still uses v1 according to https://developer.carbonblack.com/reference/enterprise-response/6.1/rest-api/#process-search